src/rpc/blockchain.cpp

```diff
@@ -370,6 +370,20 @@ static UniValue getdifficulty(const JSONRPCRequest& request)
     return GetDifficulty(::ChainActive().Tip());
 }
 
+static UniValue RBFTransactionStateToString(RBFTransactionState state)
+{
+    switch (state) {
+    case RBFTransactionState::UNKNOWN:
+        throw std::logic_error("unreachable code");
+    case RBFTransactionState::REPLACEABLE_BIP125:
+        return "bip125";
+    case RBFTransactionState::FINAL:
+        return NullUniValue; // not replaceable
+        // no default case, so the compiler can warn about missing cases
+    }
+    assert(false);
```

Comment:
Maybe with #17192 this would be updated to something like:

```c++
    switch (state) {
    case RBFTransactionState::REPLACEABLE_BIP125:
        return "bip125";
    case RBFTransactionState::FINAL:
        return NullUniValue; // not replaceable
    case RBFTransactionState::UNKNOWN:
        break; // should never happen
    }
    CHECK_NONFATAL(false);
```

Reply:
Indeed. Will adjust the second of the pulls that gets merged accordingly.